src/main/java/org/cafesip/sipunit/processing/RequestProcessingResult.java

```diff
@@ -0,0 +1,38 @@
+package org.cafesip.sipunit.processing;
+
+/**
+ * Contains provisional data on whether a request event was accepted by any {@link RequestProcessingStrategy} and if
+ * that processing step was successful.
+ * <p>
+ * Created by TELES AG on 15/01/2018.
+ *
+ * @see RequestProcessor
+ */
+public class RequestProcessingResult {
```

Comment:
Thinking about converting this to an enum, because the double boolean tautology may confuse the implementing user of the strategies

Reply:
@teles-mami makes sense

Reply:
After further analysis, we will need this form for the event processing logic. 

Specifically in "presence" event processing, we will have successful (orphaned) events which will be processed later on, but not in the current handler.

Leaving the logic as-is.